lighthouse-core/gather/connections/connection.js

```diff
@@ -109,6 +109,11 @@ class Connection {
   }
 
   _handleRawError(object, callback) {
+    // We proactively disable a few domains. Ignore any errors
+    if (object.error.message && object.error.message.includes('DOM agent hasn\'t been enabled')) {
```

Comment:
The catch() in the driver is after we ended up here as this is immediately after the error is returned over the connection.

this will only suppress errors with this message string.  (So the "any errors" comment should probably say "Ignore the DOM.disable error.")